src/Symfony/Component/Validator/Resources/translations/validators.sv.xlf

```diff
@@ -224,7 +224,7 @@
             </trans-unit>
             <trans-unit id="59" resname="This is not a valid International Bank Account Number (IBAN).">
                 <source>This value is not a valid International Bank Account Number (IBAN).</source>
-                <target state="needs-review-translation">Detta värde är inte ett giltigt Internationellt Bankkontonummer (IBAN).</target>
+                <target>Värdet är inte ett giltigt Internationellt Bankkontonummer (IBAN).</target>
```

Comment:
Let's do lower case. It is not a brand name. 

```suggestion
                <target>Värdet är inte ett giltigt internationellt bankkontonummer (IBAN).</target>
```

(Wikipedia agrees too: https://sv.wikipedia.org/wiki/Internationellt_bankkontonummer)